src/harness/testcases/testdata/device_a_exz.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
No need to create this, you can just move existing device_a into the zone. If we did this for every test case, we'd have a data explosion.

Reply:
Removed device_x_exz.json files. Changes done as per comment

Reply:
Seems like the file is still here? Or am I misunderstanding?